db/health/migrate/20230614191047_change_health_instrument_race_to_json.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeHealthInstrumentRaceToJson < ActiveRecord::Migration[6.1]
+  def change
+    safety_assured { change_column :hca_assessments, :race, 'jsonb USING to_json(ARRAY[race])' }
+    safety_assured { change_column :pctp_careplans, :race, 'jsonb USING to_json(ARRAY[race])' }
```

Comment:
What were these before?  Maybe this needs to be an `up` not `change` method?  It doesn't look reversible to me

Reply:
Yeah the enclosing function should be `up`, they were varchar before